TestModels/Constructor/runtimes/rust/src/client/get_constructor.rs

```diff
@@ -1,19 +1,20 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+// Do not modify this file. This file is machine generated, and any changes to it will be overwritten.
 impl super::Client {
     /// Constructs a fluent builder for the [`GetConstructor`](crate::operation::get_constructor::builders::GetConstructorFluentBuilder) operation.
     ///
     /// - The fluent builder is configurable:
-    ///   - [`value(impl Into<String>)`](crate::operation::get_constructor::builders::GetConstructorFluentBuilder::value) / [`set_value(Option<String>)`](crate::operation::get_constructor::builders::GetConstructorFluentBuilder::set_value):<br>required: **false**<br>(undocumented)<br>
+    ///   - [`value(impl Into<Option<::std::string::String>>)`](crate::operation::get_constructor::builders::GetConstructorFluentBuilder::value) / [`set_value(Option<::std::string::String>)`](crate::operation::get_constructor::builders::GetConstructorFluentBuilder::set_value): (undocumented)<br>
     /// - On success, responds with [`GetConstructorOutput`](crate::operation::get_constructor::GetConstructorOutput) with field(s):
-    ///   - [`internal_config_string(Option<String>)`](crate::operation::get_constructor::GetConstructorOutput::internal_config_string): (undocumented)
-    ///   - [`blob_value(Option<Blob>)`](crate::operation::get_constructor::GetConstructorOutput::blob_value): (undocumented)
-    ///   - [`boolean_value(Option<bool>)`](crate::operation::get_constructor::GetConstructorOutput::boolean_value): (undocumented)
-    ///   - [`string_value(Option<String>)`](crate::operation::get_constructor::GetConstructorOutput::string_value): (undocumented)
-    ///   - [`integer_value(Option<i32>)`](crate::operation::get_constructor::GetConstructorOutput::integer_value): (undocumented)
-    ///   - [`long_value(Option<i64>)`](crate::operation::get_constructor::GetConstructorOutput::long_value): (undocumented)
+    ///   - [`blob_value(Option<::aws_smithy_types::Blob>)`](crate::operation::get_constructor::GetConstructorOutput::blob_value): (undocumented)
```

Comment:
Minor point (not in scope for this PR), but it might be a bit better to emit things in the original smithy model order rather than sorting them.